src/main/scala/no/gdl/bookapi/model/api/Book.scala

```diff
@@ -72,6 +72,7 @@ case class Book(@(ApiModelProperty@field)(description = "The id of the book") id
                 @(ApiModelProperty@field)(description = "A UUID for this book") uuid: String,
                 @(ApiModelProperty@field)(description = "The title of the book") title: String,
                 @(ApiModelProperty@field)(description = "Description of the book") description: String,
+                @(ApiModelProperty@field)(description = "Indicates the language this book has been translated from") translatedFrom: Option[Language],
```

Comment:
Why is this an optional field? Isn't a translation always translated _from_ a certain language?

Reply:
In some cases (like imported books) we don't know the original language of the book, and some books are in their original language. So a None means that either this book is originally in the specified language or we don't know from which language it was translated.